androidApp/src/main/java/co/yml/ycharts/app/presentation/WaveChartActivity.kt

```diff
@@ -0,0 +1,45 @@
+package co.yml.ycharts.app.presentation
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import co.yml.kmm.charts.CommonMainScreen
+import co.yml.ycharts.app.R
+import co.yml.ycharts.app.ui.compositions.AppBarWithBackButton
+import co.yml.ycharts.app.ui.theme.YChartsTheme
+
+class WaveChartActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            YChartsTheme {
+                Scaffold(modifier = Modifier.fillMaxSize(),
+                    backgroundColor = YChartsTheme.colors.background,
+                    topBar = {
+                        AppBarWithBackButton(
+                            stringResource(id = R.string.title_wave_chart),
+                            onBackPressed = {
+                                onBackPressed()
+                            })
+                    })
+                {
+                    Box(
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .padding(it),
+                        contentAlignment = Alignment.TopCenter
+                    ) {
+                        CommonMainScreen(chartType = 2)
```

Comment:
Can we name it chart type here?

Reply:
Addressed the comment in https://github.com/yml-org/YCharts/pull/74 PR